test/common/packagelib.py

```diff
@@ -87,6 +88,13 @@ def setUp(self):
             self.machine.execute(f"mkdir -p {empty_repo_dir} || true")
             self.machine.execute(f"repo-add {empty_repo_dir}/empty.db.tar.gz")
             self.machine.write("/etc/pacman.conf", config)
+            # Clean up possible leftover lockfile
+            self.machine.execute("""
```

Comment:
I'd either do this *before* the (original) restore_dir(), or as addCleanup during teardown. I'd prefer the latter, but no strong opinion.